themes.gitconfig

```diff
@@ -94,6 +96,7 @@
 [delta "tangara-chilensis"]
     # author: https://github.com/clnoll
     commit-decoration-style = bold box ul "#34fd50"
+    dark = true
     file-style = omit
     file-decoration-style = none
     hunk-header-style = file line-number syntax
```

Comment:
Would it make sense to add a sentence to the instructions at the top of this file explaining that authors should include (exactly) one of the light or dark bool flags?

Reply:
Don't worry, I've pushed a commit doing this.